challenge.md

```diff
@@ -34,4 +47,12 @@
 
 Create a script where you declare a variable with a student's data
 (name, age and photo URL). Next, get the elements from the HTML
-and fill them in with the student's information.
\ No newline at end of file
+and fill them in with the student's information.
+
+`const Name = document.querySelector("h2.student-name");`
+`const years = document.querySelector("span.student-age");`
+`const img = document.querySelector("img.student-photo").src="maxresdefault.jpg";`
+`const Student_Name = document.createTextNode("Manel");`
+`const Student_years = document.createTextNode("18");`
+`Name.appendChild(Student_Name);`
+`years.appendChild(Student_years);`
```

Comment:
Great implementation!
Nevertheless, it' good to keep order in code implementations.
1st: define constants
2nd: define selectors
3rd: modify properties.

Here you have a simpler and more ordered example:
```javascript
const studentName = "John Doe";
const studentAge = "20";
const studentPhoto =
  "https://gravatar.com/avatar/54d2f681b52a49307e305224f9e6ac51?s=400&d=robohash&r=x";

const $name = document.querySelector(".student-name");
const $age = document.querySelector(".student-age");
const $photo = document.querySelector(".student-photo");

$name.textContent = studentName;
$age.textContent = studentAge;
$photo.src = studentPhoto;
$photo.alt = studentName;

```